kernel/aster-nix/src/thread/task.rs

```diff
@@ -31,6 +32,7 @@ pub fn create_new_user_task(user_space: Arc<UserSpace>, thread_ref: Weak<Thread>
             user_mode.context().syscall_ret()
         );
         loop {
+            enable_irq_if_disabled();
```

Comment:
We should never leave the kernel in a random state where the state of the IRQs is not even known. So this change is also a red flag that implies something wrong happens in the kernel.